test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -219,6 +220,92 @@ public async Task ParentInstanceId_Not_Assigned_In_Orchestrator(bool extendedSes
             }
         }
 
+        private static Mock<INameResolver> GetNameResolverMock((string Key, string Value)[] settings)
+        {
+            var mock = new Mock<INameResolver>();
+            foreach (var setting in settings)
+            {
+                mock.Setup(x => x.Resolve(setting.Key)).Returns(setting.Value);
+            }
+
+            return mock;
+        }
+
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task WritesToConsole()
+        {
+            var expectedOutput = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
+            string orchestratorName = nameof(TestOrchestrations.SayHelloInline);
+
+            // To capture console output in a StringWritter
+            using (StringWriter sw = new StringWriter())
+            {
+                // Set console to write to StringWritter
+                Console.SetOut(sw);
+                var nameResolver = new SimpleNameResolver(new Dictionary<string, string>()
+                {
+                    { "CONTAINER_NAME", "val1" },
+                    { "WEBSITE_STAMP_DEPLOYMENT_ID", "val3" },
+                    { "WEBSITE_HOME_STAMPNAME", "val4" },
+                });
+
+                // Run trivial orchestrator
+                using (var host = TestHelpers.GetJobHost(
+                    this.loggerProvider,
+                    nameResolver: nameResolver,
+                    testName: "CanWriteToConsole",
+                    enableExtendedSessions: false,
+                    storageProviderType: "azure_storage"))
+                {
+                    await host.StartAsync();
+                    var client = await host.StartOrchestratorAsync(orchestratorName, input: "World", this.output);
+                    var status = await client.WaitForCompletionAsync(this.output);
+                    await host.StopAsync();
+                }
+
+                this.output.WriteLine(sw.ToString());
+
+                // Ensure the console included some basic data
+                Assert.Contains(expectedOutput, sw.ToString());

```

Comment:
A bit more validation would be nice. I.e. splitting into newlines and validating that the JSON contains the correct values for the new columns.

Reply:
I agree that this could use a bit more validation.

So, I can (and will) validate that the output is a valid JSON-parseable string. I can also validate some minimal columns that are always present (EventId, TimeStamp, Tid, Pid). _Does that sound sufficient_? Beyond that, I worry that this may be overly specific, especially considering that our logs are so noisy and irregular in shape.

I'm **always** in favor of more testing, just want to also be mindful of our deadline here.

Reply:
Sounds good to me!